src/sass/app/trade.scss

```
@@ -641,6 +642,18 @@
                 .payout_wrapper {
                     margin-right: 20px;
                 }
+                @media screen and (max-width: 527px) {
```

What do you think about refactoring amount_wrapper_div to mobile first design like this:

```suggestion
                .amount_wrapper_div {
                display: flex;
                flex-wrap: wrap;
                flex-direction: column;
                align-items: center;
                margin-bottom: 10px;
  
                &, .contract_amount, .contract_payout {
                    vertical-align: middle;
                }
                
                div {
                    flex-grow: 1;
                }
                
                @media screen and (max-width: 527px) {
                  max-width: 100%;
                  flex-grow: 1; 
                }
                
                @media screen and (min-width: 528px) {
                  flex-direction: row;

                  .stake_wrapper {
                    margin-left: 20px;
                  }
                  .payout_wrapper {
                    margin-right: 20px;
                  }
                }
}
```

---

@hojjat-fs sorry for long time answering. Decided to not to refactoring this code because refactoring only deriv-com in typescript. Thanks for curious point of view, it's useful